yarn.lock

```diff
@@ -12018,9 +12018,9 @@ __metadata:
   linkType: hard
 
 "caniuse-lite@npm:^1.0.0, caniuse-lite@npm:^1.0.30001109, caniuse-lite@npm:^1.0.30001196, caniuse-lite@npm:^1.0.30001313":
-  version: 1.0.30001387
-  resolution: "caniuse-lite@npm:1.0.30001387"
-  checksum: 4da02e20c3932632a6fd9d0bfef172da8edc2936c524e47877746e4028d5a22f41994f9749b64b4ee23ac864fdbf5e3dc8664bd874ad016bdda018428545d38d
+  version: 1.0.30001458
```

Comment:
Not sure if having yarn.lock merged is bad or not. Please let me know if I need to remove this from the commit.

Reply:
I do need some suggestion on what to do about the yarn.lock conflict. I did not mean to include the yarn.lock in this commit.

Reply:
You should add a new commit removing this file from your changes. 

[This](https://stackoverflow.com/a/46571620) should do the trick. It must be something like(untested):

> git checkout origin/trunk -- yarn.lock

Reply:
Thank you, @WBerredo! Worked like a charm.